Tools/clinic/libclinic/parse_args.py

```diff
@@ -437,6 +446,20 @@ def parse_one_arg(self) -> None:
             parser_code = libclinic.normalize_snippet(parsearg, indent=4)
             self.parser_body(parser_code)
 
+    def parse_varpos_only(self) -> None:
```

Comment:
Just to be sure: using `METH_O_PROTOTYPE` is the reason why the `_impl` suffix was removed in some functions, right?

Reply:
Right. For example, currently `set.update()` uses METH_FASTCALL and creates a tuple in the generated code that calls the `*_impl()` function. Now it uses METH_VARARGS and calls the implementation directly.